cmd/kubeadm/app/cmd/config.go

```diff
@@ -307,6 +308,8 @@ func NewCmdConfigImagesPull() *cobra.Command {
 		},
 	}
 	AddImagesCommonConfigFlags(cmd.PersistentFlags(), cfg, &featureGatesString)
+	AddImagesPullFlags(cmd.PersistentFlags(), &cfgPath)
```

Comment:
you should bind this to `cfg.CRISocket` instead

Reply:
all fixed